fe/fe-core/src/main/java/com/starrocks/catalog/Catalog.java

```diff
@@ -4220,13 +4220,16 @@ private void createHiveTable(Database db, CreateTableStmt stmt) throws DdlExcept
     }
 
     public static void getDdlStmt(Table table, List<String> createTableStmt, List<String> addPartitionStmt,
-                                  List<String> createRollupStmt, boolean separatePartition, boolean hidePassword) {
-        getDdlStmt(null, table, createTableStmt, addPartitionStmt, createRollupStmt, separatePartition, hidePassword);
+                                  List<String> createRollupStmt, boolean separatePartition,
+                                  boolean hidePassword, boolean hideAggregateTypeName) {
+        getDdlStmt(null, table, createTableStmt, addPartitionStmt, createRollupStmt,
+                separatePartition, hidePassword, hideAggregateTypeName);
     }
 
     public static void getDdlStmt(String dbName, Table table, List<String> createTableStmt,
                                   List<String> addPartitionStmt,
-                                  List<String> createRollupStmt, boolean separatePartition, boolean hidePassword) {
+                                  List<String> createRollupStmt, boolean separatePartition,
+                boolean hidePassword, boolean hideAggregateTypeName) {
```

Comment:
Fix line-wrapping style

Reply:
It is enough to handle the primary key type in this function, no need to use hideAggregateTypeName parameter.